cleanupBuckets.js

```diff
@@ -68,97 +62,96 @@ function _getKeys(keys) {
 }
 
 // delete all versions of an object
-function _deleteVersions(bucket, objectsToDelete, cb) {
-    // multi object delete can delete max 1000 objects
+async function _deleteVersions(bucket, objectsToDelete) {
+    if (!objectsToDelete || objectsToDelete.length === 0) {
```

Comment:
This is a bit redundant with the check already in `cleanupVersions`? I think we should be able to merge them